docs/topics/Permissions.md

```diff
@@ -219,3 +221,7 @@ Roles without colors (`color == 0`) do not count towards the final computed colo
   "mentionable": false
 }
 ```
+
+## Permissions For Timed Out Members
+
+Timed out members will temporarily lose all permissions except `VIEW_CHANNEL` and `READ_MESSAGE_HISTORY`. Owners and admin users with `ADMINISTRATOR` permissions are exempt.
```

Comment:
```suggestion
Timed out members will temporarily lose all permissions except `VIEW_CHANNEL` and `READ_MESSAGE_HISTORY`. Owners and users with `ADMINISTRATOR` permissions are exempt.
```

I think admin users is too much

Reply:
technically you're right but maybe in this case we can leave it just to make it extra clear, since devs are familiar with the term "admin user" we use elsewhere :)